### DISCUSSION TITLE
Help in implementing staggered scheme

### DISCUSSION THREAD

**Discussion by ayush4424** [idaholab/moose#27044] [Q&A Modules: Phase field]
Can anyone help how to implement this staggered scheme in moose ?
![Screenshot 2024-03-07 121530](https://github.com/idaholab/moose/assets/135033854/0d0e5d35-0607-4b5b-b35c-682a260265f1)
I tried using multi-app but it gives divergence errors when i try to solve steady-state problems using Transient.
Any help would be appreciated.
https://doi.org/10.1016/j.tafmec.2022.103348
This is the link of the document from which screenshot is taken.
Regards,
Ayush

**Comment by GiudGiud** [comment]
Hello

A multiapp is a good way to do this.
You ll want to use a transient multiapp if you want to perform relaxation to steady state transients. 

The main way to debug this is to freeze one solve with Problem/solve=false and see if the solution for the other one is correct

Guillaume
